FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_DNS.c

```diff
@@ -376,6 +399,17 @@ typedef struct xDNSAnswerRecord DNSAnswerRecord_t;
 		char pcName[ 1 ];
 	} DNSCallback_t;
 
+	static portINLINE ipDECL_CAST_PTR_FUNC_FOR_TYPE( DNSCallback_t )
+	{
+		/* coverity[misra_c_2012_rule_11_3_violation] */
+		return ( DNSCallback_t *)pvArgument;
+	}
+	static portINLINE ipDECL_CAST_CONST_PTR_FUNC_FOR_TYPE( DNSCallback_t )
```

Comment:
Delete lines 407-412 as this isn't used.

Reply:
Deleted.